### ISSUE TITLE
Can I train a new model based on the parameter of codebert-base for a new programming languange?

### ISSUE THREAD

**Issue by A7um** [microsoft/CodeBERT#213]
Hi,
Thanks for the excellent work. I want to train a new model based on the parameter of codebert-base to make it available for a new programming language. 
My question is: Currently I cannot get the parameters of RTD and MLM head. Therefore, I cannot calculate the loss of these two objectives.  In this situation, how can I train the new model?

Thanks

**Comment by guoday** [comment]
You can use [this](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling) to continue pre-training CodeBERT-base for a new programming language.

**Comment by A7um** [comment]
> You can follow [this](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling) to continue pre-training CodeBERT-base for a new programming language.

This link is only for the MLM objective, Is it possible to pre-train the RTD objective?

**Comment by guoday** [comment]
Sorry, we can't provide pre-training code for CodeBERT. In fact, the improvement of RTD objective is limited and more complicated. I suggest you can just use MLM objective to continue pre-training.

**Comment by A7um** [comment]
Nice, thanks for your suggestion, and thanks again for the excellent work.
